tests/generator/test_tunnels.py

```diff
@@ -699,6 +699,146 @@ def test_vxlan(self):
 [ipv4]
 method=link-local
 
+[ipv6]
+method=ignore\n'''})
+
+    def test_vxlan_maclearning_arpproxy_shortcircuit_true(self):
+        self.generate('''network:
+  renderer: %(r)s
+  version: 2
+  ethernets:
+    eth0: {}
+  tunnels:
+    vxlan1005:
+      link: eth0
+      mode: vxlan
+      id: 1005
+      mac-learning: true
+      arp-proxy: true
+      short-circuit: true''' % {'r': self.backend})
+
+        if self.backend == 'networkd':
+            self.assert_networkd({'vxlan1005.netdev': '''[NetDev]
```

Comment:
I think we should make some use of the `ND_*` templates from `generator/base.py` here, e.g. ND_VLAN, to avoid repeating ourselves too much. Maybe concatenate or str.replace()  relevant pieces of the string as needed.